src/containers/Settings.js

```diff
@@ -2,14 +2,22 @@ import React, { Component } from 'react';
 // import { connect } from 'react-redux';
 import { Link } from 'react-router';
 
-
 export default class Settings extends Component {
```

Comment:
This isn't specific to this pull request, so feel free to disregard it for now, but I suspect this component could be stateless as it doesn't—ummm—use any state.